duecredit/injections/mod_sklearn.py

```diff
@@ -9,7 +9,7 @@
 """Automatic injection of bibliography entries for numpy module
 """
 
-from ..entries import Doi, BibTeX
+from ..entries import Doi, BibTeX, Url
```

Comment:
Yes but we do need it because some references in sklearn are just a URL

On Sun, Aug 9, 2015 at 7:39 PM, Yaroslav Halchenko <notifications@github.com
> wrote:

> In duecredit/injections/mod_sklearn.py
> <https://github.com/duecredit/duecredit/pull/42#discussion_r36595371>:
>
> > @@ -9,7 +9,7 @@
> >  """Automatic injection of bibliography entries for numpy module
> >  """
> >
> > -from ..entries import Doi, BibTeX
> > +from ..entries import Doi, BibTeX, Url
>
> Url seems ended up being unused, right?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/duecredit/duecredit/pull/42/files#r36595371>.
>